### ISSUE TITLE
Create official facets for controller battery level

### ISSUE THREAD

**Issue by Frooxius** [Yellow-Dog-Man/Resonite-Issues#2174]
### Describe the Underlying Issue

With the new facet anchors there's more space for users to customize their UI and add additional pieces of information. We should provide more official facets for common things.

One of the things people like to have is their controller battery status.

### Expected Behavior

- Create a set of facets that display the current controller battery level in a nice way.
- They should also properly indicate when the controllers are charging
- They should handle gracefully when the information isn't available (the battery status will be smaller than 0)
- Split the facets between left & right controllers in case users want to put each one on the corresponding hand

**Comment by shadowpanther** [comment]
I'd also request the headset charge level/status as a facet too. But I'm not sure whether we already have that as a value somewhere.

**Comment by epicEaston197** [comment]
Will these be put onto the facet hand anchors by default?

**Comment by Frooxius** [comment]
I'm not sure, we probably leave these for people to customize on their own to their liking.

**Comment by ModernBalloonie** [comment]
I would like an official headset battery level status facet!

**Comment by shiftyscales** [comment]
We have the GeneralHeadset node which indicates battery level, so that should be possible to do as well. @AegisTheWolf @RyuviTheViali @RueShejn

**Comment by ExoTheWicker** [comment]
Quest controllers would't work those as I belive the oculus API doesn't share that info to SteamVR, altough steamlink might.

**Comment by shadowpanther** [comment]
> Quest controllers would't work those as I belive the oculus API doesn't share that info to SteamVR, altough steamlink might.

Well, the facets would currently work for people using other controllers. And as soon as the controller charge level would be passed from Quest to SteamVR, it'd also work for Quest users.

**Comment by RyuviTheViali** [comment]
Okay, I have created three new facets, two for the batteries of one's left and right controllers, and one for the battery level of one's headset. These facets currently have placeholder icons for the left/right controller and headset. The facets will change color the lower the batter level gets, they'll be white if charging, and purple if no battery information is available for that device.

The facets also auto-orient vertically or horizontally if the facet is tall or wide respectively. 

I've placed these facets in `Resonite Essentials/Facets/Templates` and they are named `LeftControllerBattery`, `RightControllerBattery`, and `HeadsetBattery`.

![Image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/15852289/00aee265-d570-415b-b482-d8b213c08d41)
